installation/cmd/run.sh

```diff
@@ -39,21 +58,61 @@ do
 done
 set -- "${POSITIONAL[@]}" # restore positional parameters
 
+MINIKUBE_ARGS="--domain \"${DOMAIN}\" --vm-driver \"${VM_DRIVER}\""
+
+if [[ $KNATIVE ]]; then
```

Comment:
This is a very shortsighted solution.
Think about introducing more feature flags
The number of combinations (if/else) grows exponentially:
```
if(knative)
  if(other) {}
    else {}
  fi
else
  if(other) {}
    else {}
  fi
fi
```
If you have three feature flags, the number of combinations is eight, and it can't be handled by if/else anymore.
This has to be refactored so that adding more ffs is possible in the future.